profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
+  "status": "production",
+  "status_since": "2012-01-01",
+  "type": "Generic",
```

Comment:
That's again always a difficult label. :wink: We tend to say that [SlipStream](http://sixsq.com/slipstream) is a **PaaS** to directly manage **IaaS** in order to deliver **SaaS**. Therefore I've chosen `Generic`.
What so you think?

@mebster
Any input from your side?